src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -334,6 +334,31 @@ public static void ACos_Advanced(double real, double imaginary, double expectedR
             VerifyRealImaginaryProperties(result, expectedReal, expectedImaginary);
         }
 
+        public static IEnumerable<object[]> ACos_Legacy_TestData ()
+        {
+            // Boundary values
+            yield return new object[] { double.MaxValue, 0, double.NaN, double.NaN };
+            yield return new object[] { double.MinValue, 0, double.NaN, double.NaN };
+
+            // Invalid values
+            foreach (double invalidReal in s_invalidDoubleValues) {
```

Comment:
VS2017 now has support for specifying much of this via the editorconfig file (powered by Roslyn for the C#/VB code style options). It just isn't used many places yet.